fw/rbcx-coprocessor/include/Motor.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "utils/Debug.hpp"
+#include "utils/Regulator.hpp"
+
+#include <algorithm>
+#include <stdint.h>
+
+inline const uint16_t motorLoopFreq = 100;
+
+enum class MotorState {
```

Comment:
Když ten enum dáš dovnitř `class Motor` pod `private:`, tak nemusíš pak psát to `MotorState::`, a je jasné, že ten enum je jen pro tuhle třídu, viz https://github.com/RoboticsBrno/RB3204-RBCX/blob/master/fw/rbcx-coprocessor/include/Esp32Manager.hpp#L24